playbooks/roles/rabbitmq/defaults/main.yml

```diff
@@ -17,9 +17,9 @@ RABBIT_USERS:
   - name: 'admin'
     password: "{{ RABBIT_ADMIN_PASSWORD }}"
   - name: 'edx'
-    password: 'edx'
+    password: "{{ XQUEUE_RABBITMQ_PASS }}"
   - name: 'celery'
-    password: 'celery'
+    password: "{{ EDXAPP_CELERY_PASSWORD }}"
```

Comment:
And XQUEUE_RABBITMQ_PASS? I don't see this variable. BTW did you test it for devstack ? I expect some errors on devstack because we don't have server-vars there.